packages/react-widgets/src/Combobox.js

```diff
@@ -310,6 +310,7 @@ class Combobox extends React.Component {
     } = this.props
     let { accessors } = this.state
     let valueItem = accessors.findOrSelf(data, value)
+    const resovledInputId = inputProps?.id || this.inputId
```

Comment:
```suggestion
    const resolvedInputId = inputProps?.id || this.inputId
```

The logic tho still isn't quite right. the reason `inputId` is stored on the instance at all is because it's referenced in more than one place. In this case it's used in `aria-labelledby={inputId}` on the list component which would now point to a none existent id when `inputProps.id` is provided